apps/modernization-ui/src/components/Table/Actions.tsx

```diff
@@ -0,0 +1,29 @@
+import { Button } from '@trussworks/react-uswds';
+
+export const Actions = ({ handleAction }: any) => {
+    return (
+        <div className="actions-card">
+            <Button
+                onClick={() => handleAction('details')}
```

Comment:
I have actually done this on purpose because this action is called into the nested component so I don't want to call it to the components, so that's the reason to do it from the actual component